test/skimNtuple2016_HHbtag.cpp

```diff
@@ -0,0 +1,6078 @@
+          continue;
+        }
+
+        //bool bPairFound = false;
+        //int njets = jets_and_sortPar.size();
+
+        // medium WP for 2016 is: 0.6321 for DeepCSV, 0.3093 for DeepFlavor
+        //if(useDeepFlavor)
+        //{
+        //    if (jets_and_sortPar.at(njets-2).first>0.3093) bPairFound = true;
+        //}
+        //else
+        //{
+        //    if (jets_and_sortPar.at(njets-2).first>0.6321) bPairFound = true;
+        //}
+
+        //const int bjet1idx = jets_and_sortPar.at(njets-1).second ;
+        //int bjet2idx_temp  = jets_and_sortPar.at(njets-2).second ;
+
+        //VBF tag
+        int VBFidx1 = -1;
+        int VBFidx2 = -1;
+        theSmallTree.m_isVBF = 0;
+
+        // build all the possible VBF-jet pairs
+        std::vector< tuple<float, int, int> > VBFcand_Mjj;
+        if (theBigTree.jets_px->size ()>1)
+        {
+          for (unsigned int iJet = 0; (iJet < theBigTree.jets_px->size ()) && (theSmallTree.m_njets < maxNjetsSaved); ++iJet)
+          {
+            // JET PF ID cut
+            if (theBigTree.PFjetID->at (iJet) < PFjetID_WP) continue; // 0 ; don't pass PF Jet ID; 1: tight, 2: tightLepVeto
+
+            // Skip the already selected b-jets
+            if (int (iJet) == bjet1idx) continue;
+            if (int (iJet) == bjet2idx) continue;
+            //if(bPairFound && int (iJet) == bjet2idx_temp) continue;
+
+            TLorentzVector ijet;
+            ijet.SetPxPyPzE(
+                           theBigTree.jets_px->at (iJet),
+                           theBigTree.jets_py->at (iJet),
+                           theBigTree.jets_pz->at (iJet),
+                           theBigTree.jets_e ->at (iJet)
+                           );
+
+            if (ijet.DeltaR (tlv_firstLepton) < lepCleaningCone) continue ;
+            if (ijet.DeltaR (tlv_secondLepton) < lepCleaningCone) continue ;
+            if(ijet.Pt() < 30.) continue;
+            if(fabs(ijet.Eta()) > 4.7) continue; // max eta allowed: https://twiki.cern.ch/twiki/bin/view/CMS/HiggsToTauTauWorkingLegacyRun2#Jets
+
+            // Apply PUjetID only to jets with pt < 50 GeV ( https://twiki.cern.ch/twiki/bin/view/CMS/HiggsToTauTauWorkingLegacyRun2#Jets )
+            // PU jet ID WP = 2: loose
+            if (PUjetID_WP > -1)
+            {
+              if ( !(CheckBit(theBigTree.jets_PUJetIDupdated_WP->at(iJet), PUjetID_WP)) && ijet.Pt()<50.) continue;
+            }
+
+            for (unsigned int kJet = iJet+1 ;   (kJet < theBigTree.jets_px->size ()) && (theSmallTree.m_njets < maxNjetsSaved) ;  ++kJet)
+            {
+              // JET PF ID cut
+              if (theBigTree.PFjetID->at (kJet) < PFjetID_WP) continue; // 0 ; don't pass PF Jet ID; 1: tight, 2: tightLepVeto
+
+              // Skip the already selected b-jets
+              if (int (kJet) == bjet1idx) continue;
+              if (int (kJet) == bjet2idx) continue;
+              //if (bPairFound && int (kJet) == bjet2idx_temp) continue;
```

Comment:
also this comment is useless

Reply:
ok! I will remove the useless commented lines.